docs/Operators.md

```diff
@@ -2368,6 +2368,72 @@ Other versions of this operator: <a href="Changelog.md#GRU-1">GRU-1</a>
 </dl>
 
 
+#### Examples
+
+<details>
+<summary>defaults</summary>
+
+```python
+input = np.array([[[1., 2.], [3., 4.], [5., 6.]]]).astype(np.float32)
+
+input_size = 2
+hidden_size = 5
+weight_scale = 0.1
+
+node = onnx.helper.make_node(
+    'GRU',
+    inputs=['X', 'W', 'R'],
+    outputs=['Y'],
+    hidden_size=hidden_size
+)
+
+W = weight_scale * np.ones((1, 3 * hidden_size, input_size)).astype(np.float32)
```

Comment:
Can we give capture the 3 as a variable with a suitable name instead of a magic number for the benefit of the reader? Similarly for LSTM tests below we can capture 4 as a variable.

Reply:
Sure, updated the PR.